stdlib/test/memory/test_memory.mojo

```diff
@@ -132,6 +132,55 @@ def test_memcmp():
     assert_equal(errors2, 0)
 
 
+def test_memcmp_overflow():
+    print("== test_memcmp_overflow")
```

Comment:
You can remove those `print`. We don't use filecheck anymore, and the print in the other test functions are relics from another time when FileCheck was running this file.